presto-native-execution/presto_cpp/main/types/FunctionMetadata.cpp

```diff
@@ -134,6 +134,25 @@ const std::vector<protocol::TypeVariableConstraint> getTypeVariableConstraints(
   return typeVariableConstraints;
 }
 
+const std::vector<protocol::LongVariableConstraint> getLongVariableConstraints(
+    const FunctionSignature& functionSignature) {
+  std::vector<protocol::LongVariableConstraint> longVariableConstraints;
+  const auto functionVariables = functionSignature.variables();
+  for (const auto& [name, signature] : functionVariables) {
+    if (signature.isIntegerParameter()) {
+      protocol::LongVariableConstraint longVariableConstraint;
+      longVariableConstraint.name =
+          boost::algorithm::to_lower_copy(signature.name());
+      longVariableConstraint.expression =
```

Comment:
Do we know if the grammar for the Velox expressions are perfectly compatible with the Presto side ?

Reply:
Confirmed that the grammar conforms to https://github.com/facebookincubator/velox/blob/main/velox/expression/type_calculation/TypeCalculation.yy, which should work with Presto as well.